File: 02.C#-Fundamentals/Exercise-Basic Syntax, Conditional Statements and Loops/04. Print and sum.cs
namespace Basic_Syntax__Conditional_Statements_and_Loops_Exercise
{
    internal class Program
    {
        static void Main(string[] args)
        {
           int start=int.Parse(Console.ReadLine());
            int end=int.Parse(Console.ReadLine());
            int sum = 0;
            for (int i = start; i <= end; i++)
            {
                Console.Write($"{i} ");
                sum += i;
            }
            Console.WriteLine();
            Console.WriteLine($"Sum: {sum}");
        }
    }
}

File: 08.JS-Front-End/01.JS Syntax Fundamentals - Exercises/04. Print And Sum.js
function sum(start,end){
    let totalSum= 0;
    let array =[];
for (let i = start; i <= end; i++) {
    array.push(i);
    totalSum+=i;
}
console.log(array.join(' '));
console.log(`Sum: ${totalSum}`)
}
